### PR TITLE
adding touchpoints top tasks survey to site banner

### PR DESCRIPTION
**Pull request by bonnieAcameron**
## Description

Jacline and I have updated the site banner by:

1. Removing the patterns library content
2. Adding the link to the top tasks survey with custom text 
3. changing the color of the banner

## Preview link
Preview link: [Home page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/bc-jc-touchpoints-for-toptasks/)

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [ ] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [ ] Run `npm test` and make sure the tests for the files you have changed have passed.
- [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [ ] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

**Review by amyleadem — CHANGES_REQUESTED** [review]
Looking good so far! I made a few suggestions and had a couple of questions for you.

File: _layouts/landing.html [hunk 1]
```diff
@@ -4,15 +4,9 @@
 
 <main id="main-content">
   <div
-    class="bg-gold-20v padding-x-2 desktop:padding-x-4 padding-y-1 line-height-sans-3 font-lang-4 text-bold"
+    class="bg-info-light padding-x-2 desktop:padding-x-4 padding-y-1 font-lang-4"
```
Comment (amyleadem) [inline _layouts/landing.html:7]:
```suggestion
    class="bg-info-light padding-x-2 desktop:padding-x-4 padding-y-1 line-height-sans-5 font-lang-4"
```

File: _layouts/landing.html [hunk 2]
```diff
@@ -4,15 +4,12 @@
...
-    Introducing inclusive design patterns:
-    <a
-      href="{{site.baseurl}}/together/"
-      onclick="ga('send', 'event', 'Viewed Together report', 'Clicked to view Together report from inside site');"
-      class="text-ink text-normal hover:text-no-underline"
-      >Read about the inclusive design patterns project.</a
-    >
+  What are the most important things you need to accomplish on our site? Please take <a 
+  href="https://touchpoints.app.cloud.gov/touchpoints/0a9a9227/submit" 
+  >this 5 minute survey</a> so we can make the site work better for you!</a>
```
[the remark this answers is not in the document]
Comment (jaclinec) [reply _layouts/landing.html]: @amyleadem it should not be hyphenated. Thanks for catching that!

File: _layouts/landing.html [hunk 3]
```diff
@@ -4,15 +4,12 @@
 
 <main id="main-content">
   <div
-    class="bg-gold-20v padding-x-2 desktop:padding-x-4 padding-y-1 line-height-sans-3 font-lang-4 text-bold"
+    class="accent-cool padding-x-2 desktop:padding-x-4 padding-y-1 line-height-sans-3 font-lang-4 text-bold"
   >
-    Introducing inclusive design patterns:
-    <a
-      href="{{site.baseurl}}/together/"
-      onclick="ga('send', 'event', 'Viewed Together report', 'Clicked to view Together report from inside site');"
```
[the remark this answers is not in the document]
Comment (bonnieAcameron) [reply _layouts/landing.html:12]: I don't think so. I think the previous team set up some tracking, but our GA on the backend isn't really configured to accept this information meaningfully.

**Comment by amyleadem** [comment]
@jaclinec @bonnieAcameron I took the liberty of updating some of the classes on here, and I added a [preview link](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/bc-jc-touchpoints-for-toptasks/) in the description. Let me know if it looks the way you wanted!

**Review by bonnieAcameron — COMMENTED** [review]
Love them, thank you!

**Review by bonnieAcameron — COMMENTED** [review]
looks good!

**Review by amyleadem — APPROVED** [review]
Looks good to me! Just want to get confirmation on the color choice (and other minor style changes) from @mejiaj before merging. 

Checked on Safari, Chrome, Firefox, Edge for Mac

**Review by mejiaj — APPROVED** [review]

**Pull request merged**